release/models/interfaces/openconfig-if-sdn-ext.yang

```diff
@@ -47,11 +47,34 @@ module openconfig-if-sdn-ext {
         This is used by an external programming entity to disable an interface
         (usually part of an aggregate) for the purposes of forwarding
         traffic. This allows a logical aggregate to continue to be
-        used with partial capacity, for example.  Note that setting
-        `forwarding-viable = false` is not equivalent to
-        administratively disabling the interface -- in particular, the
-        interface is expected to participate in L2 protocols such as
-        LLDP or LACP even if it blocked from forwarding traffic.";
+        used with partial capacity. Setting `forwarding-viable = false` is not 
+        equivalent to administratively disabling the interface.
+        Some rules to follow when a port/bundle is set for 
```

Comment:
Here and throughout I suggest using the OpenConfig terms interface and aggregate : 

```suggestion
        Some rules to follow when an interface or aggregate interface is set for 
```